test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/013_GetVehicleData_with_deprecated_fuelLevel_and_fuelLevel_State.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
```

Comment:
Why the previous scripts about deprecated parameters(10, 12) contain only one RPC, but current case is covered for all RPCs and in separated scripts?